src/test/ui/asm/inline-syntax.rs

```diff
@@ -25,6 +25,8 @@ macro_rules! global_asm {
 
 #[lang = "sized"]
 trait Sized {}
+#[lang = "drop_in_place"]
+fn drop_in_place<T>(_: *mut T) {}
```

Comment:
Seems like removing the requirement for the lang item would remove these unnecessary test changes.